### ISSUE TITLE
Fix Accessibility issues raised by axe DevTools

### ISSUE THREAD

**Issue by drn05r** [eprints/eprints3.4#345]
An alternative Accessibility tester to the WAVE Accessibility tool is axe DevTools (https://www.deque.com/axe/devtools/).  EPrints should be tested with this tool and and resulting issues detected should be addressed appropriately.
